src/javascript/itemsC.js

```diff
@@ -110,11 +110,11 @@ window.itemF={
 		return daringRec;
 	},
 },
-
+//Room list; electronics store 0, general store 1, CLINIC store/ doctor's office(?) 2, adult store 3, fashion central 4, intimate store 5, klip klops 6, 
 window.itemsC={
 	pyjamasMale : {
 		id : "pyjamasMale",
-		name: "Comfy pajamas",
+		name: "Comfy boy's pajamas",
```

Comment:
```suggestion
		name: "Comfy pajamas",
```
Since it's in the men's store, we don't need to specify that they're male.